package.json

```diff
@@ -10,14 +10,14 @@
     "dist",
     "src"
   ],
-  "main": "./dist/npm-exports.js",
+  "main": "./dist/src/npm-exports.js",
   "sideEffects": false,
   "repository": {
     "type": "git",
     "url": "https://github.com/api3dao/airseeker-v2"
   },
   "scripts": {
-    "build": "tsc --project tsconfig.build.json",
+    "build": "rm -rf dist && tsc --project tsconfig.build.json",
```

Comment:
We should either run `pnpm run clean` or not run this at all (and only run the `pnpm run clean` in the release instructions).

Reply:
Okay I reverted change on this script and merging now. Thanks for the review 💪🏼